3.evaluate_model/scripts/nbconverted/get_LOIO_probabilities.py

```diff
@@ -41,21 +50,22 @@
 print(f"There are {num_images} images to perform LOIO evaluation on per model.")
 
 
+# ### Get LOIO probabilities
+# 
+
 # In[4]:
 
 
 # directory to load the models from
-models_dir = pathlib.Path("../2.train_model/models/")
+models_dir = pathlib.Path("../2.train_model/models/multi_class_models")
```

Comment:
```suggestion
models_dir = pathlib.Path("../2.train_model/models/multi_class_models").reslove(strict=True)
```
This may be out of scope for this PR. Recently the `pathlib.Path.resolve()` has been really usefull in handling different paths (i.e symlink paths). The `strict` parameters checks if the provided path exists before conducting any down stream functions. 

Here's the [documentation](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve), may be useful for future implementations for path handling!

Reply:
I think it is probably outside the scope of this PR, but thank you for letting me know about this function.